app/grandchallenge/reader_studies/urls.py

```diff
@@ -92,9 +85,39 @@
         name="example-ground-truth",
     ),
     path(
-        "<slug>/display-sets/add/",
+        "<slug>/display-sets/create/",
         AddDisplaySetsToReaderStudy.as_view(),
-        name="add-displaysets",
+        name="display-sets-create",
+    ),
+    path(
+        "<slug>/display-sets/create-single/",
+        AddDisplaySetToReaderStudy.as_view(),
+        name="display-set-create",
+    ),
+    path(
+        "<slug>/display-sets/<uuid:pk>/",
+        DisplaySetDetail.as_view(),
+        name="display-set-detail",
+    ),
+    path(
+        "<slug>/display-sets/<uuid:pk>/update/",
+        DisplaySetUpdate.as_view(),
+        name="display-set-update",
+    ),
+    path(
+        "<slug>/display-sets/new/interfaces/create/",
+        DisplaySetInterfacesCreate.as_view(),
+        name="display-set-new-interfaces-create",
+    ),
+    path(
+        "<slug>/display-sets/<uuid:pk>/interface/create/",
+        DisplaySetInterfacesCreate.as_view(),
+        name="display-set-interfaces-create",
+    ),
+    path(
+        "<slug>/display-sets/<uuid:pk>/interfaces/<interface_pk>/files/update/",
```

Comment:
Interface slugs should be used, not pks:

```suggestion
        "<slug>/display-sets/<uuid:pk>/interfaces/<slug:interface_slug>/files/update/",
```